spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/store/channel/AbstractChannelMessageStoreQueryProvider.java

```diff
@@ -23,6 +23,7 @@
  *
  * @since 2.2
  */
+@Deprecated
```

Comment:
Please, also add a `@deprecated in favor of default methods in ChannelMessageStoreQueryProvider` into a JavaDoc of this class after `@since`